### PR TITLE
#167313354 Write end-to-end tests for the application

### PR DESCRIPTION
**Pull request by nkalyesubula**
#### What does this PR do?

- This is intended to add configurations and tests for End to End and unit testing

#### #### Description of Task to be completed?

- Write end-to-end tests for the application

#### How should this be manually tested?

- Clone the repo, and then run **npm run test** for unit testing and npm **run test:cypress** for end to 
end testing

#### What are the relevant pivotal tracker stories?
[#167313354](https://www.pivotaltracker.com/story/show/167313354)

#### Screenshots (if appropriate)
End-to-End testing screenshot
Unit-testing screenshot

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+          },
+        ],
+      },
+    ],
+  },
+  plugins: [
```
Comment (habinezadalvan) [inline webpack.config.js]: @nkalyesubula, why does it seem like you did @Anguandia's work?

Comment (nkalyesubula) [reply webpack.config.js]: Fixed, please check again. It was a problem  caused after squashing .

**Review by habinezadalvan — APPROVED** [review]

**Review by minega25 — APPROVED** [review]
LGTM

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Comment by SokoPaulSokool** [comment]
Goodjob. You should rebase and also squash your commits

**Pull request closed without merging**